PharoTour/PharoTour.pillar

```diff
@@ -353,7 +352,7 @@ The ""System Browser"", also known as "Class Browser", is one of the key tools
 used for programming. As we shall see, there are several interesting browsers
 available for Pharo, but this is the basic one you will find in any image.
 
-@@todo TODO: factorial on number would be better. and should redo the screenshot
+@@authorToDo TODO: factorial on number would be better. and should redo the screenshot
```

Comment:
@DamienCassou - what does the `@@authorToDo` tag do? 
In any case, can we standardize on opening Github issues (for todo items) instead of leaving them in-text?